### ISSUE TITLE
LNURL channel crashed the app.

### ISSUE THREAD

**Issue by danielbroll** [ZeusLN/zeus#1908]
### Describe the bug

I scanned a lnurl channel QR code.

The first LNURL request succeeds and the 'please confirm' page is shown in the app.

![4E6564CA-97E4-44EF-89C4-8C8F199E9CFF](https://github.com/ZeusLN/zeus/assets/98091744/6ebbe675-7b57-4329-b208-06259ce45341)

Pressing connect minimized the the app.

No second LNURL request arrives on our backend, no channel is opened :(
Tried it multiple times always the same behavior.

At the end I was able to request a new channel by first connecting the embedded node manually through the app and then performing the lnurl protocol manually through my browser.

### Reproduce

This is an LNURL channel for **testnet** on our platform: LNURL1DP68GURN8GHJ7AMFV3NK2APWWD6XZEMFDENJUUR0VD4K2ARZD96XXMMFDCHXXMMD9ASHQ6F0D3H82UNV9A3KSCTWDEJKCTEKVVCKVVE4VGUJ6DTPXYCJ6DP48YMJ6WPNVVCZ6VECV9JKZDTXVVEKGDMZVSP8TV

```
{"uri":"031630330c3104d416f05e7fc33681603518373d1db0488a870c775edb062bfec3@34.65.45.157:9737","callback":"https://api.staging.pocketbitcoin.com/api/lnurl/channel/6c1f35b9-5a11-4597-83c0-38aea5fc3d7b/callback","k1":"none","tag":"channelRequest"}
```

If you tell me you your node pubkey, I can configure it on our platform so opening a new channel to your node is possible.

### ZEUS version

v0.8.0

### Node interface

Embedded LND

### Network

Tor

### Device

iphone 12

### Device operating system

iOS 16.1.1

**Comment by kaloudis** [comment]
@danielbroll can you add 02758997f184be06f4350b136db0bed6f8af27cf732576db66d311dade77ba3c53 first?

**Comment by danielbroll** [comment]
@kaloudis configured, you can give it a try!

**Comment by kaloudis** [comment]
No problem testing against LND REST.

@danielbroll can you add `0390f55c5db50399b0e710f9d6ce8ef4a606277d4e86160970d6dc526edcc8f087` - this is a testnet embedded node

**Comment by kaloudis** [comment]
Sorted. https://github.com/ZeusLN/zeus/commit/38eeb69ce3732dcbf39e1c0226d0c8a501e7dcd0#diff-344f815ec93887638f9b1d02d5b756c5a2feb3db78b6bf6782b5a9ef91fecc58L136

We were using node URIs to submit the node's pubkey to the service, and since the embedded node doesn't have any exposed URIs, it crashed. Created a new getter to fetch the pubkey appropriate.

Greatly appreciate the bug fix and the support. We'll roll this out in the next `v0.8.1` build

**Comment by danielbroll** [comment]
Thank you for providing the fix!

**Comment by kaloudis** [comment]
Fix shipped in v0.8.1
